src/api/Item.js

```diff
@@ -279,6 +285,27 @@ class Item extends Base {
         }
     };
 
+    /**
+     * Validate an item update request
+     *
+     * @param {string|void} itemID - ID of item to share
+     * @param {BoxItemPermission|void} itemPermissions - Permissions for item
+     * @return {Error|null}
```

Comment:
I am not sure if returning `null` here makes sense when validated successfully. I think `true` may make more sense, since the specified item is indeed valid. 

Perhaps it should `throw` when invalid, using the values from those functions. Conforming to existing patters should override this suggestion.

Reply:
@booc0mtaco I updated the code to reuse the `catch` blocks for invalid requests. Could you review again? 🙏